package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "bas-platform",
 	"displayName": "App-Studio-Toolkit Extension",
 	"description": "SAP Business Application Studio Toolkit extension",
-	"version": "0.0.33",
+	"version": "0.0.13",
```

Comment:
Is it by matter?

Reply:
Yes. Later we will decide both when to bump it to 1.0.0